src/cmd/Makefile.am

```diff
@@ -68,7 +68,8 @@ dist_fluxcmd_SCRIPTS = \
 	flux-cron \
```

Comment:
Commit message spelling "implementaition"

Reply:
Thanks, fixed and force-pushed.